src/event.rs

```diff
@@ -44,9 +46,16 @@ pub struct EventProxyWinit<T: 'static> {
     inner: EventLoopProxy<T>,
 }
 
-impl<T> EventProxy<T> for EventProxyWinit<T> {
+impl<T> EventProxy<T> for EventProxyWinit<T>
+where
+    T: Debug,
+{
     fn send_event(&self, event: T) {
-        self.inner.send_event(event);
+        // Not sure if unwrap is the best idea here, but I don't know enough
+        // about the app to say other wise
+        //
+        // Feel free to replace
+        self.inner.send_event(event).unwrap();
```

Comment:
I'd not recommand unwrap here, because it's not an issue that should panic.

In the long term, I think it would be a better idea to send the `Result` to the application layer and handle them there.